tests/unit/db/database_tests.py

```diff
@@ -696,6 +696,57 @@ def test_get_query_result_without_kwargs(self):
             ['julia001', 'julia002', 'julia003', 'julia004']
         )
 
+    def test_get_query_result_without_fields(self):
+        """
+        Test that retrieving the QueryResult for a query works as expected when
+        executing a query
```

Comment:
I think it would be nice if this comment said what the expected behaviour is. Maybe add something like:
> Assert that the QueryResult docs include all expected fields when no fields parameter is provided.